starter-arbitrary-uid/bin/uid_entrypoint

```diff
@@ -1,8 +1,6 @@
 #!/bin/sh
-if [ -w /etc/passwd ]; then 
-  USER_ID=$(id -u)
-  id ${USER_ID} &> /dev/null
-  if [ $? -ne 0 ]; then
+if [ -w /etc/passwd ]; then
+  if ! whoami &> /dev/null; then
```

Comment:
this _should_ work but `USER_ID=$(id -u)` line needs to stay i suspect.  the variable is required for the sed.